doc/manual/src/package-management/garbage-collector-roots.md

```diff
@@ -1,7 +1,7 @@
 # Garbage Collector Roots
 
 The roots of the garbage collector are all store paths to which there
-are symlinks in the directory `prefix/nix/var/nix/gcroots`. For
+are symlinks in the directory prefix `/nix/var/nix/gcroots`. For
```

Comment:
You can configure a prefix to those paths but that is not typical done and it is probably more confusing prefixing those paths. Same in line 15.

Reply:
This seems to be an instance of https://github.com/NixOS/nix/issues/7771

I wouldn't exactly say this means documentation is "correct" because it's not helpful or self-explanatory that way. Would be great if we could address the issue systematically.